packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -127,13 +128,23 @@ const renderHeaderContent = ({
   service,
   script,
   startTime,
+  timezone,
+  locale,
 }) => {
   const isOnDemand = state === 'onDemand';
   const isLive = state === 'live';
   const hiddenTextProps = stateLabel === 'Live' ? { lang: 'en-GB' } : {};
 
   const labelWrapperProps = isLive ? { 'aria-hidden': 'true' } : {};
 
+  const formattedStartTime = formatUnixTimestamp(
+    startTime,
+    'HH:mm',
```

Comment:
I can't quite remember, we said we're formatting everything as HH:mm for now and then looking at the LT format later on?

Reply:
Yes, that was what we agreed on :)